data.json

```diff
@@ -1236,69 +1248,57 @@
     "issueNumber": 634
   },
   {
-    "name": "Saad Irfan",
-    "githubUsername": "msaaddev",
-    "imageUrl": "https://user-images.githubusercontent.com/44341551/128627907-fbeb7a8a-f0d7-454f-a077-cf10c8876aaa.png",
-    "issueNumber": 636
-  },
-  {
-    "name": "Pratham Bhagat",
-    "githubUsername": "bhagatpratham",
-    "imageUrl": "https://user-images.githubusercontent.com/66031743/128864527-177a97f6-967c-4b18-93d2-c02472274612.PNG",
-    "issueNumber": 639
-  },
-  {
-    "name": "Gunjan Paneri",
-    "githubUsername": "iamgunjan",
-    "imageUrl": "https://user-images.githubusercontent.com/53567221/129094169-e012e3fb-3ba9-47e7-9173-0e09e7517034.png",
-    "issueNumber": 642
+    "name": "Vivek Singh",
+    "githubUsername": "vivmost",
+    "imageUrl": "https://user-images.githubusercontent.com/58567847/92334655-ad8e1600-f0ad-11ea-9de9-c44ee1ff7420.jpg",
+    "issueNumber": 81
   },
   {
-    "name": "Vasudev Jaiswal",
-    "githubUsername": "VasudevJaiswal",
-    "imageUrl": "https://user-images.githubusercontent.com/85981735/129255772-5bac726e-d98e-4eb4-8c9f-c2206b72c976.png",
-    "issueNumber": 641
+    "name": "Yajush Vyas",
+    "githubUsername": "geekyvyas",
+    "imageUrl": "https://user-images.githubusercontent.com/35025375/98467874-c924b280-21fd-11eb-810b-2d17e0342e10.png",
+    "issueNumber": 163
   },
   {
-    "name": "Fayas Noushad",
-    "githubUsername": "FayasNoushad",
-    "imageUrl": "https://user-images.githubusercontent.com/76828314/129467637-18e9f157-4038-4aed-83e7-7d55e239c03f.jpg",
-    "issueNumber": 646
+    "name": "Yash Indane",
+    "githubUsername": "YashIndane",
+    "imageUrl": "https://user-images.githubusercontent.com/53041219/99989860-22353e80-2dd9-11eb-95c5-07dd04eecd7d.png",
+    "issueNumber": 180
   },
   {
-    "name": "Shubham Kukreti",
-    "githubUsername": "KukretiShubham",
-    "imageUrl": "https://user-images.githubusercontent.com/51391473/129485691-4a1249b6-3203-4f15-ac8c-654a67d50ee4.png",
-    "issueNumber": 648
+    "name": "Yash Sehgal",
+    "githubUsername": "yashsehgal",
+    "imageUrl": "https://user-images.githubusercontent.com/62352288/113475867-6aeeb980-9495-11eb-86be-bfa0085a6f48.png",
+    "issueNumber": 429
   },
   {
-    "name": "Harkirat Singh",
-    "githubUsername": "harkiratsm",
-    "imageUrl": "https://user-images.githubusercontent.com/71957674/130051242-25747856-913e-4613-9cc6-7384dc58d752.png",
-    "issueNumber": 645
+    "name": "afc163",
```

Comment:
These names don't have the first letter capitalized, so that's why they appear at the bottom. Please consider fixing it manually, keeping it without being capitalized. :)

Reply:
Done!